src/pages/LoginPage/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,59 @@
+import template from './LoginForm.hbs';
+
+import Button from '../../../components/Button/Button';
+import Input from '../../../components/Input/Input';
+import Logo from '../../../components/Header/Logo/Logo'
+import urls from '../../../router/urls'
+import Link from '../../../components/Header/Link/Link'
+import { router } from '../../../router/Router'
+import { login } from '../../../api/api'
+
+class LoginForm {
+  constructor(parent) {
+    this.parent = parent;
+  }
+
+  asHTML() {
+    return template(this.display);
+  }
+
+  displayErrorOrRedirect(response) {
+    if (response.error == null) {
+      localStorage.setItem('username', response.User.username);
```

Comment:
сделайте на бэке лучше так, чтобы все было с маленькой буквы - выглядит оч странно)

```suggestion
      localStorage.setItem('username', response.user.username);
```